sdks/python/test-suites/dataflow/py2/build.gradle

```diff
@@ -48,7 +52,7 @@ task preCommitIT(dependsOn: ['sdist', 'installGcpTest']) {
     ]
     def cmdArgs = project.mapToArgString([
         "test_opts": testOpts,
-        "sdk_location": "${project.buildDir}/apache-beam.tar.gz",
+        "sdk_location": files(configurations.distTarBall.files).singleFile,
```

Comment:
This line actually can be removed for all similar integration tests sine the default `sdk_location` (defined in [run_integration_test.sh](https://github.com/apache/beam/blob/master/sdks/python/scripts/run_integration_test.sh#L72)) is pointing to the correct location. However, I left it here since people who read this configuration will know how it's set and related to `:sdks:python:sdist`. I'm open to discussion if you think it can be removed.

Reply:
I'm afraid not. We need full path of this tarball, but `configurations.distTarBall` returns a string `configuration ':sdks:python:test-suites:dataflow:py2:distTarBall'`